classes/Search.php

```diff
@@ -204,23 +204,20 @@ public static function find(
         foreach ($words as $key => $word) {
             if (!empty($word) && strlen($word) >= (int)Configuration::get('PS_SEARCH_MINWORDLEN')) {
                 $word = str_replace(array('%', '_'), array('\\%', '\\_'), $word);
+                $start_pos = (int)$word[0] == '-';
```

Comment:
start_pos is only used on line 210, put it next to it

Reply:
Change applied